client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/components/GetHelpPage.tsx

```diff
@@ -0,0 +1,324 @@
+        <Close />
+      </IconButton>
+    </div>
+  );
+};
+
+const Bubble: FC<{ children: ReactNode }> = ({ children }) => (
+  <div className="rounded-lg p-2 max-w-[70%] bg-gray-300">{children}</div>
+);
+
+const MessageList: FC<{
+  messages: LiveFeedbackMessage[];
+  editorRef: RefObject<EditorRef>;
+  focusedMessageIndex: number | undefined;
+  loading: boolean;
+}> = ({ messages, editorRef, focusedMessageIndex, loading }) => {
+  const listRef = useRef<HTMLUListElement>(null);
+
+  useEffect(() => {
+    if (listRef.current && focusedMessageIndex !== undefined) {
+      const messageElement = listRef.current.children[
+        focusedMessageIndex
+      ] as HTMLElement;
+      if (messageElement) {
+        listRef.current.scrollTop = messageElement.offsetTop;
+      }
+    }
+  }, [messages, focusedMessageIndex]);
+
+  const focusEditorOnFeedbackLine = (linenum: number | null): void => {
+    if (typeof linenum !== 'number' || linenum < 0) return;
+    editorRef.current?.editor?.gotoLine(linenum, 0);
+    editorRef.current?.editor?.selection?.setAnchor(linenum - 1, 0);
+    editorRef.current?.editor?.selection?.moveCursorTo(linenum - 1, 0);
+    editorRef.current?.editor?.focus();
+  };
+
+  return (
+    <List ref={listRef} className="flex-grow overflow-auto pb-16">
+      {messages.map((message, index) => (
+        <ListItem
+          key={index}
+          className={`py-0 ${
+            message.sender === 'Codaveri' ? 'justify-start' : 'justify-end'
+          }`}
+          onClick={() => focusEditorOnFeedbackLine(message.linenum)}
+        >
+          <ListItemText
+            className={`rounded-lg p-2 max-w-[70%] flex-none ${
+              message.sender === 'Codaveri' ? 'bg-gray-300' : 'bg-blue-100'
+            }`}
+            primary={
+              message.isBold ? (
+                <Typography className="text-[1.3rem]" fontWeight="bold">
+                  {message.text}
+                </Typography>
+              ) : (
+                <Typography className="text-[1.3rem]">
+                  {message.text}
+                </Typography>
+              )
+            }
+            secondary={
+              message.timestamp
+                ? moment(message.timestamp).format(SHORT_DATE_TIME_FORMAT)
+                : ''
+            }
+          />
+        </ListItem>
+      ))}
+      {loading && (
+        <ListItem className="justify-start py-0">
+          <Bubble>
+            <BeatLoader color="grey" size={8} />
+          </Bubble>
+        </ListItem>
+      )}
+    </List>
+  );
+};
+
+const SuggestionButtons: FC<{
+  loading: boolean;
+  suggestions: string[];
+  handleSendMessage: (message: string) => void;
+}> = ({ loading, suggestions, handleSendMessage }) => (
+  <div className="absolute top-[-4rem] flex justify-center mb-0 w-full flex-row h-[52%] overflow-x-auto">
+    {suggestions.map((suggestion, index) => (
+      <Button
+        key={index}
```

Comment:
Don't use the index as the `key`, the translation object itself should have an id.

Reply:
Actually, to make it extensible, suggestions can just take in an array of strings as you have done so (since the API would just give you an array of strings).  
But don't use the index as key, can just use the suggestion itself as the key.